### DISCUSSION TITLE
Add Quality Options To Fallback Audio Options.

### DISCUSSION THREAD

**Discussion by HDMOVIESOURCE** [RandomEngy/VidCoder#1101] [Ideas]
Hi,

Would it be possible to add bit-rate control and options to the fallback audio? What I like to do is set my audio to passthrough for all types of audio if available. If no passthrough is available I then set it to AC3. However, when you do this Videcoder just picks some random bit-rate. I would like an option to always use 640 kbps on the fallback audio encoder and to have compression and volume controls like you would when actually picking AC3 with no passthrough.

This would entirely automate everything. It would passthrough Dolby Atmos if a track existed, and if a track had 6000 kbps PCM track it would then revert to my AC3 settings.

This possible?
Thanks.

**Comment by grossmro** [comment]
Hello RandomEngy,

This is also a question I do have. I want to digitalize my dashcam and smartphone videos as well as the videos from family.

I want to passthrough the following codecs MP2/MP3/AAC/AC3/EAC3 and fallback all other codecs to AC3, as these codecs should recognized and handled by any device.
But there are no settings... How can I define the settings for the fallback code? As bitrate, sampling rate, channels etc.?

Am I missing something, or how and where can I do this (or define the used default values)?

Thanks and kind regards

**Reply by RandomEngy** [reply]
I've had a few people ask for this over the years. It's a good idea and on the list of stuff to work on when I get some time.

**Reply by HDMOVIESOURCE** [reply]
Thank you.
